content/efcore/getting-started/toc.md

```diff
@@ -0,0 +1,2 @@
+- [Introduction to Entity Framework Core](what-is-ef-core.md)
```

Comment:
Abbreviate Entity Framework in the table of contents please. I requested this change in PR #19 (1_getting_started_readme) as well.